src/gui/optionsdlg.ui

```diff
@@ -1769,6 +1769,19 @@
                 </layout>
                </item>
                <item>
+                <widget class="QPushButton" name="banListButton">
+                 <property name="sizePolicy">
+                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+                   <horstretch>0</horstretch>
+                   <verstretch>0</verstretch>
+                  </sizepolicy>
+                 </property>
+                 <property name="text">
+                  <string>Open the list of manually banned IP addresses</string>
```

Comment:
OK I understand. Then feel free to add address/addresses to my proposed string changes.